### PR TITLE
feat : index page for screens

### PR DESCRIPTION
**Pull request by mohammed-sas**
Added landing page for screens
csb : https://m2k9j.csb.app/

Screen Shot 2022-02-05 at 11 29 08 AM

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    
+}
+ol{
+    background: var(--white);
+    border: 1px solid var(--purple-700);
+    padding: 3rem;
+    width: 20rem;
+}
+ol li{
+    margin: 5px 0;
```
Comment (gadgetfather) [inline index.css:23]: you can add hover effect list items so the user will know where exactly he is interacting.

**Review by gadgetfather — APPROVED** [review]

File: index.css (hunk 1 above)
Comment (mohammed-sas) [reply index.css:23]: sure , thank you for the suggestion

**Review by kart-c — APPROVED** [review]
Looks good, easy navigation to all screens

**Comment by mohammed-sas** [comment]
> Looks good, easy navigation to all screens

thank you

**Pull request merged**